scipy/fft/_backend.py

```diff
@@ -86,6 +96,19 @@ def register_backend(backend):
     ------
     ValueError: If the backend does not implement ``numpy.scipy.fft``.
 
+    Examples
+    --------
+    We can register a new fft backend:
+
+    >>> from scipy.fft import fft, register_backend, set_global_backend
+    >>> class NoopBackend:  # define an invalid Backend
+    ...     __ua_domain__ = "numpy.scipy.fft"
+    ...     def __ua_function__(self, func, args, kwargs):
+    ...          return NotImplemented
+    >>> set_global_backend(NoopBackend())  # set the invalid backend as global
```

Comment:
This seems to break the other doctests. I think you will need to restore the default backend at the end of the example:
```python
set_global_backend("scipy")
```

Reply:
Oh, OK. I added it.